contexts/credentials/v2

```diff
@@ -5,6 +5,28 @@
 
     "id": "@id",
     "type": "@type",
+    
+    "kid": {
+      "@id": "https://www.iana.org/assignments/jwt#kid",
+      "@type": "@id"
+    },
+    "iss": {
+      "@id": "https://www.iana.org/assignments/jwt#iss",
+      "@type": "@id"
+    },
```

Comment:
Compare to how we define `issuer` today, this JSON-LD maps behavior for `iss` the same way.